pkg/controller/pod_test.go

```diff
@@ -171,12 +187,26 @@ var _ = Describe("Dynamic Attachment controller", func() {
 					)
 					Eventually(<-eventRecorder.Events).Should(Equal(expectedEventPayload))
 				})
+
+				It("the pod network-status no longer features the removed network", func() {
+					Eventually(func() []nad.NetworkStatus {
+						updatedPod, err := k8sClient.CoreV1().Pods(namespace).Get(context.TODO(), podName, metav1.GetOptions{})
+						if err != nil {
+							return nil
+						}
+						status, err := networkStatus(updatedPod.Annotations)
+						if err != nil {
+							return nil
+						}
+						return status
+					}).Should(BeEmpty())
```

Comment:
One question, `BeEmpty` does not pass on `nil` right ? because is this is the case we can enmasquerade an error.

Reply:
Done.